agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2118,13 +2121,29 @@ bool backhaul_manager::handle_1905_1_message(ieee1905_1::CmduMessageRx &cmdu_rx,
     }
     case ieee1905_1::eMessageType::MULTI_AP_POLICY_CONFIG_REQUEST_MESSAGE: {
         return handle_multi_ap_policy_config_request(cmdu_rx, src_mac);
+    case ieee1905_1::eMessageType::AP_METRICS_QUERY_MESSAGE: {
+        return handle_ap_metrics_query(cmdu_rx, src_mac);
     }
     default: {
         // TODO add a warning once all vendor specific flows are replaced with EasyMesh
         // flows, since we won't expect a 1905 message not handled in this function
         return false;
     }
     }
+    }
+}
+bool backhaul_manager::handle_slave_1905_1_message(ieee1905_1::CmduMessageRx &cmdu_rx,
+                                                   const std::string &src_mac)
+{
+    switch (cmdu_rx.getMessageType()) {
+    case ieee1905_1::eMessageType::AP_METRICS_RESPONSE_MESSAGE: {
+        return handle_ap_metrics_response(cmdu_rx, src_mac);
+    }
+    default: {
+        LOG(DEBUG) << "Unknown 1905 message";
```

Comment:
also would be great to add the message type for the error log.
```suggestion
        LOG(DEBUG) << "Unknown 1905 message:" << cmdu_rx.getMessageType()
```